scripts/cve-reports/send-scan.py

```diff
@@ -100,13 +109,12 @@ def parse_sarif(filename):
                 "priority": severity_to_priority_map.get(severity, "Lowest"),
             }
         )
-
     return record_list
 
 
-def main(report_path, jira_url):
+def main(report_path, jira_url, gh_meta=False, verbose=False):
     input_path = Path(report_path)
-
+    gh_metadata = get_github_meta() if gh_meta else None
```

Comment:
See my suggestion down below.

```suggestion
    gh_metadata = get_github_meta() if gh_meta else {}
```